tfhe/benches/shortint/glwe_packing_compression.rs

```diff
@@ -0,0 +1,68 @@
+use criterion::{black_box, criterion_group, Criterion};
+use rayon::iter::{IntoParallelIterator, ParallelIterator};
+use tfhe::shortint::glwe_compression::COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS;
+use tfhe::shortint::prelude::*;
+
+fn glwe_packing(c: &mut Criterion) {
+    #[allow(clippy::single_element_loop)]
+    for (name, param, comp_param, number_to_pack) in [(
+        "1",
+        PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+        COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS,
+        256,
+    )] {
+        let bench_name = "shortint-glwe-packing";
+
+        let mut bench_group = c.benchmark_group(bench_name);
+
+        // Generate the client key and the server key:
+        let (cks, _sks) = gen_keys(param);
```

Comment:
could just call cks = ClientKey::new(param), I don't think the compiler can delete the code for the sks because of side effects in thread locals

Reply:
Done